osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/config/NotificationClientConfig.java

```diff
@@ -1,126 +1,29 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
 package org.opensmartgridplatform.adapter.ws.core.application.config;
 
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.PropertySource;
-import org.springframework.context.annotation.PropertySources;
-import org.springframework.core.io.FileSystemResource;
-import org.springframework.oxm.jaxb.Jaxb2Marshaller;
-import org.springframework.ws.soap.saaj.SaajSoapMessageFactory;
-import org.springframework.ws.soap.security.support.KeyStoreFactoryBean;
-
-import org.opensmartgridplatform.adapter.ws.clients.SendNotificationServiceClient;
 import org.opensmartgridplatform.adapter.ws.core.application.services.NotificationServiceWsCore;
 import org.opensmartgridplatform.adapter.ws.shared.services.NotificationService;
 import org.opensmartgridplatform.adapter.ws.shared.services.NotificationServiceBlackHole;
-import org.opensmartgridplatform.shared.application.config.AbstractConfig;
-import org.opensmartgridplatform.shared.infra.ws.DefaultWebServiceTemplateFactory;
+import org.opensmartgridplatform.adapter.ws.shared.services.PublicLightingNotificationClientConfigBase;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 
 @Configuration
-@PropertySources({ @PropertySource("classpath:osgp-adapter-ws-core.properties"),
-        @PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true),
-        @PropertySource(value = "file:${osgp/AdapterWsCore/config}", ignoreResourceNotFound = true), })
-public class NotificationClientConfig extends AbstractConfig {
-
-    // Notification web service global properties.
-
-    @Value("${web.service.notification.enabled:true}")
-    private boolean webserviceNotificationEnabled;
-
-    @Value("${web.service.notification.security.enabled:false}")
-    private boolean webserviceNotificationSecurityEnabled;
-
-    @Value("${web.service.notification.url:http://localhost:8080/web-api-net-management/soap/osgp/notificationService/}")
-    private String webserviceNotificationUrl;
-
-    @Value("${jaxb2.marshaller.context.path.notification:org.opensmartgridplatform.adapter.ws.schema.core.notification}")
-    private String marshallerContextPathNotification;
-
-    @Value("${apache.client.max.connections.per.route:20}")
-    private int maxConnectionsPerRoute;
-
-    @Value("${apache.client.max.connections.total:400}")
-    private int maxConnectionsTotal;
-
-    // Notification web service application name, user name and organization
-    // properties.
-
-    @Value("${web.service.notification.application.name:WS_CORE}")
```

Comment:
Should the application name be set in the properties as the default value is no longer component specific?

Reply:
Done